haystack/nodes/connector/crawler.py

```diff
@@ -49,28 +47,28 @@ class Crawler(BaseComponent):
 
     def __init__(
         self,
-        output_dir: str,
         urls: Optional[List[str]] = None,
         crawler_depth: int = 1,
         filter_urls: Optional[List] = None,
-        overwrite_existing_files=True,
         id_hash_keys: Optional[List[str]] = None,
         extract_hidden_text=True,
         loading_wait_time: Optional[int] = None,
+        output_dir: Union[str, Path, None] = None,
+        overwrite_existing_files=True,
+        save_file_path_on_meta: Optional[bool] = None,
+        file_path_meta_field_name: str = "file_path",
```

Comment:
Question: with `output_dir` we use the presence of the parameter to represent whether we should save on disk or not. Should we do the same here, removing `save_file_path_on_meta` and using  the presence of `file_path_meta_field_name` to determine whether or not saving the path in meta?

Reply:
We could do it, but we would remove a standardized name for the field when the user decides to make use of it. Do you think it would be better to let the full responsibility (turning on the feature and choosing the name of meta) with the user?